lib/matplotlib/tests/test_mlab.py

```diff
@@ -1,1074 +0,0 @@
-from numpy.testing import (assert_allclose, assert_almost_equal,
```

Comment:
My suggestion was to keep this as well (with a global pytestmark to filter warnings everywhere in the file), but I guess the Axes tests essentially cover what we want too?

Reply:
Yep, sorry, I forgot I had to "add" it...